app/src/main/java/me/saket/dank/utils/ColorReplicationIcons.kt

```diff
@@ -0,0 +1,37 @@
+package me.saket.dank.utils
+
+import android.content.Context
+import androidx.core.content.ContextCompat
+import me.saket.dank.R
+import me.saket.dank.widgets.span.CenterAlignedImageSpan
+import net.dean.jraw.models.VoteDirection
+
+object ColorReplicationIcons {
+  @JvmStatic fun pushIcon(context: Context, builder: Truss, sizeDimenResId: Int,
+                          drawableResId: Int, tintColor: Int, lineSpacingExtra: Int) {
```

Comment:
not sure what the project policy is but annotating these parameters with `@DrawableRes`, `@ColorRes` and `@DimenRes` would probably not be a bad idea.

Reply:
I'm voting for including these :)

Reply:
Sorry for such a long delay, I'll address it in a separate PR